http/query.go

```diff
@@ -372,10 +366,10 @@ func decodeQueryRequest(ctx context.Context, r *http.Request, svc influxdb.Organ
 		}
 	}
 
-	switch hv := r.Header.Get(PreferHeaderKey); hv {
-	case PreferNoContentHeaderValue:
+	switch hv := r.Header.Get(query.PreferHeaderKey); hv {
+	case query.PreferNoContentHeaderValue:
 		req.PreferNoContent = true
-	case PreferNoContentWErrHeaderValue:
+	case query.PreferNoContentWErrHeaderValue:
 		req.PreferNoContentWithError = true
 	}
```

Comment:
These cyclic dependencies are unfortunate. Ideally we'd have a the http related constants in the `http` package. If there is no easy way to break that, we can proceed with this PR and identify the problem areas in a future PR.

Reply:
Yeah, I agree 👍 

The cycle is `query -> http -> authorizer -> task -> query` and I don't see any obvious way to break that for now 😓 

It seemed easy, because `authorizer` depends on `task/backend` only for constants, but when I removed them, the cycle only changed...